Scheduler/Notification.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Scheduler
+{
+	/// <summary>
+	/// Notification Table Info
+	/// </summary>
+	public class Notification
+	{
+		public long NotificationId { get; set; }
+
+		[ForeignKey("Task")]
```

Comment:
The same as for Task table. Can you do the same without attributes?
Also this attribute will force ef to create column named "Task" and I suggest it's better to keep it's name as "TaskId", so this attribute is redundant here. Without this attribute ef core conventions will do the same and create the same relation one to many. Ef core looks for navigation properties to create relations.

Also it's good practice to make navigation property and corresponding id property stay together after all other properties:
```
public class Notification
{
    ...

    public long? TaskId { get; set; }
    public virtual Task Task { get; set; }
}